cloud/installation/poller-remove.md

```diff
@@ -0,0 +1,18 @@
+---
+id: poller-remove
+title: Removing a poller from your architecture
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import MoveResources from './_move-resources.mdx';
+
```

Comment:
```suggestion

In order to ensure the stability and integrity of your monitoring environment, please be advised that the deletion of the central poller is not recommended. The central poller plays a critical role in the overall operation of the Centreon platform. Removing it can lead to unintended disruptions in service. We are in the process of implementing safeguards to prevent the accidental deletion of the central poller.

```